src/apic-extension/azext_apic_extension/aaz/latest/apic/api/_create.py

```diff
@@ -49,6 +49,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="The name of the API.",
             required=True,
             fmt=AAZStrArgFormat(
+                pattern="^[a-zA-Z0-9-]{3,90}$",
```

Comment:
The 2024-03-01 spec provides pattern for related parameter, I think we can keep it.